setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup
+
+setup(# package information
+      name="stackclub",
+      version="0.1",
+      author="Phil Marshall",
+      author_email="dr.phil.marshall@gmail.com",
+      description="Utilities for use in e.g the Stack Club LSST tutorial notebooks",
+      long_description=open("README.md").read(),
```

Comment:
Should this be README.rst for beavis?

Reply:
Oh - I _think_ this refers to the top level package README, which is still markdown (for now). I think it can stay that way too, unless we think its just better to have only rst README files throughout (for sanity?)